esmero/format_strawberryfield
# Issue 91 into RC1 For Giancarlo to test on Drupal 9

Weirdest Pull title ever. This is intended as a way of bringing a few features together that are already in 1.0.0-RC1 but do not live in  ISSUE-91 (the one with WebAnnotations) So @giancarlobi can test in D9 all together.

How to test? 
- Once Merged (and conflict solved)
  - checkout and pull
  - Enabled the new Permissions at least for the Admin and Web Annotations Viewing for Anonymous
  - Go to any View Mode that is using the Media Formatter (OpenSeadragon), edit the Formatter settings and check the "Annotations" checkbox. Update and Save
 - Using "Shift + Mouse" draw a box in any Image driven by Openseadragon. Add more, edit (delete per item is not working yet), answer, reply. Once done, press "Edit tab" you should see a new message in any form saying you have un persisted  Annotations. "Save" to keep them, or press "Discard" and continue having a blast.

@giancarlobi @alliomeria
